### DISCUSSION TITLE
react-select in controller

### DISCUSSION THREAD

**Discussion by andrconstruction** [react-hook-form/react-hook-form#3043] [Q&A]
All work as expected

![image](https://user-images.githubusercontent.com/7435330/94718741-d3c86e00-035a-11eb-881f-1e7f674a3de6.png)

But on render I got a warning :
A component is changing a controlled input of type hidden to be uncontrolled. Input elements should not switch from controlled to uncontrolled (or vice versa).

How to resolve this warning?

**Comment by marcusflat** [comment]
Could you show the code of the Select component? Probably, the problem could be there.

**Reply by andrconstruction** [reply]
It is native react-select /  import Select from "react-select"; (https://react-select.com)

**Reply by marcusflat** [reply]
Can you provide [CSB](https://codesandbox.io/) ?

**Reply by bluebill1049** [reply]
Thanks @marcusflat for helping :)

**Reply by andrconstruction** [reply]
Problem was with null default value in react-select. You can close this question. Thanks to all

**Reply by andrconstruction** [reply]
Unfortunantly problem back again, I can reproduce it

codesandbox : https://codesandbox.io/s/sparkling-surf-z9rzc?file=/src/App.js

but now problem with checkbox/
And reset is not working if select option scomming later
